From 6b5ea10376ea3ef5e8dc3476a04d4b97816c5380 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 24 Jul 2014 18:16:12 -0400
Subject: [fs] nfs: Remove 2 unused variables

Message-id: <1406225773-16395-10-git-send-email-steved@redhat.com>
Patchwork-id: 86641
O-Subject: [RHEL7.1 PATCH 09/10] NFS: Remove 2 unused variables
Bugzilla: 1120728
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>

From: Trond Myklebust <trond.myklebust@primarydata.com>

Cc: Weston Andros Adamson <dros@primarydata.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id: aafe37504c70954fc104c88d9d15d553572dae69
Bugzilla: 1120728

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfs/direct.c b/fs/nfs/direct.c
index 4ff545f..d133d9c 100644
--- a/fs/nfs/direct.c
+++ b/fs/nfs/direct.c
@@ -913,14 +913,12 @@ static void nfs_direct_write_completion(struct nfs_pgio_header *hdr)
  spin_unlock(&dreq->lock);
 
  while (!list_empty(&hdr->pages)) {
-  bool do_destroy = true;
 
   req = nfs_list_entry(hdr->pages.next);
   nfs_list_remove_request(req);
   if (request_commit) {
    kref_get(&req->wb_kref);
    nfs_mark_request_commit(req, hdr->lseg, &cinfo);
-   do_destroy = false;
   }
   nfs_unlock_and_release_request(req);
  }
diff --git a/fs/nfs/write.c b/fs/nfs/write.c
index 461156e..30dcd0b 100644
--- a/fs/nfs/write.c
+++ b/fs/nfs/write.c
@@ -901,7 +901,6 @@ static void nfs_write_completion(struct nfs_pgio_header *hdr)
 {
  struct nfs_commit_info cinfo;
  unsigned long bytes = 0;
- bool do_destroy;
 
  if (test_bit(NFS_IOHDR_REDO, &hdr->flags))
   goto out;
@@ -927,7 +926,6 @@ remove_req:
 next:
   nfs_unlock_request(req);
   nfs_end_page_writeback(req);
-  do_destroy = !nfs_write_need_commit(hdr);
   nfs_release_request(req);
  }
 out:
-- 
1.7.1